buildSrc/src/main/java/org/elasticsearch/gradle/info/GlobalBuildInfoPlugin.java

```diff
@@ -131,6 +133,11 @@ public void apply(Project project) {
         project.getGradle().getTaskGraph().whenReady(graph -> logGlobalBuildInfo());
     }
 
+    private String formatJavaVendorDetails(JvmInstallationMetadata runtimeJdkMetaData) {
+        JvmVendor vendor = runtimeJdkMetaData.getVendor();
```

Comment:
Should we keep JvmVendor here and use `JvmVendorSpec.X` for comparison in `fips.gradle` ? Unless `runtimeJavaDetails` will also be used for descriptive purposes somewhere else in the build output

Reply:
I would keep `JvmVendor` an implementation detail here as it is actual an internal class of gradle at the moment and not use it in the `BuildParams` API